src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss

```diff
@@ -0,0 +1,8 @@
+.MyAccountPasswordForm-Button {
+    @include mobile {
+        width: 100%;
+    }
+    position: relative;
```

Comment:
Separate all your style blocks with new lines. Make sure there is an additional new-line in the end.